PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,7 @@ Please include a summary of the change with relevant motivation and context
 
 ## Linked Issues
 
-- (issue)
+- https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
```

Comment:
do we need this?

note we can't link issues from other repositories

Reply:
it has a list of key words to tell github what to do with the linked issue

Reply:
Shouldn't it be closes by default

Reply:
done